src/markdown/common.ts

```diff
@@ -20,3 +20,55 @@ export const uniqueSlug = (slug: string, slugs) => {
     slugs[uniq] = true;
     return uniq;
 };
+
+export const findEndMarker = (str: string, startPos: number = 0, beginMarker: string = "{", endMarker: string = "}", onlyEnd = false) => {
```

Comment:
a few comments in this function to understand the user facing impact would be great